tests/e2e.rs

```diff
@@ -16,18 +16,27 @@ mod jwt_vc;
 async fn w3c_vc_did_client_direct_post() {
     let (wallet, verifier) = jwt_vc::wallet_verifier().await;
 
+    // let
+
+    // let presentation_definition = PresentationDefinition::new(uuid::Uuid::new_v4())
+    //     .with_input_descriptors(input_descriptors)
+    //     .with_name(name)
+    //     .with_purpose(purpose)
+    //     .with_format(format)
+
```

Comment:
TODO: Complete e2e test case updates using the new interface. Work to be completed in #8